indico/modules/rb/client/js/common/rooms/selectors.js

```diff
@@ -137,3 +137,19 @@ export const getBuildings = createSelector(
     return Array.from(buildings).sort((a, b) => parseInt(a, 10) - parseInt(b, 10));
   }
 );
+
+export const getLocations = createSelector(
+  getAllRooms,
+  allRooms => {
+    const locations = Object.values(allRooms)
+      .map(({locationName, locationId}) => {
+        return {
+          name: locationName,
+          id: locationId,
+        };
+      })
```

Comment:
Since the callback just returns something you can use the expression-style arrow function:
```suggestion
      .map(({locationName, locationId}) => ({
        name: locationName,
        id: locationId,
      }))
```